internal/chart/chart.go

```diff
@@ -32,7 +32,6 @@ type Config struct {
 }
 
 type Release struct {
-	Flags     map[string]interface{}
```

Comment:
What's wrong with `Flags ` in release struct?
I think it's a good place to keep it here as other values connected with chart are also here.

Reply:
If the property be named `override` it might be even more natural to keep it here.

Reply:
We need have one source of truth. Now we added FlagBuilder and Flags was second source (in this case we should sunchronize both, but we don't want this) :(